src/config/Tiles.jsx

```diff
@@ -29,6 +30,10 @@ const messagesTiles = defineMessages({
     id: 'video',
     defaultMessage: 'Video',
   },
+  maps: {
```

Comment:
We don't want this tile to be default, so please add it as default customTile (we will probably have to change that to "secondary level tiles" or something along those lines, but let's leave it for the next sprint. So Google map will appear as a secondary addable level tile.